fbw-a380x/src/systems/instruments/src/FCU/Managers/AutopilotManager.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2023-2024 FlyByWire Simulations
+// SPDX-License-Identifier: GPL-3.0
+
+import { EventBus, Instrument } from '@microsoft/msfs-sdk';
+import { TemporaryHax } from './TemporaryHax';
+
+// FIXME port to MSFS avionics framework style
```

Comment:
Shall we create a GH issue so we remember this in the future?

Reply:
https://github.com/flybywiresim/aircraft/issues/9056